src/librustc_middle/ty/context.rs

```diff
@@ -2731,6 +2731,6 @@ pub fn provide(providers: &mut ty::query::Providers<'_>) {
     providers.has_panic_handler = |tcx, cnum| {
         assert_eq!(cnum, LOCAL_CRATE);
         // We want to check if the panic handler was defined in this crate
-        tcx.lang_items().panic_impl().map_or(false, |did| did.is_local())
+        tcx.lang_items().panic_impl().require().map_or(false, |did| did.is_local())
```

Comment:
Seems like this could be a method on `LangItemRecord`, `is_local` or something.

Reply:
That’s a really good point - thanks